healthcheck/core.go

```diff
@@ -531,6 +539,9 @@ func (s *Server) manager() {
 			for id := range configs {
 				if s.healthchecks[id] == nil {
 					hc := NewCheck(s.notify)
+					if s.config.DryRun {
```

Comment:
Instead of adding it to NewCheck in the cost of changing the signature, I followed the existing pattern of Check.Blocking() and added the helper function for the dryrun. Thought it's a better tradeoff this way. What do you think?